### DISCUSSION TITLE
mlflow.evaluate: custom_metrics returns overlapped graphs

### DISCUSSION THREAD

**Discussion by khojarohan4** [mlflow/mlflow#6425] [Q&A] [answered]
Not sure if this is faced by anyone yet but I have added 4 custom plots to my mlflow.evaluate that I'm passing to the custom_metrics parameter and the output on the mlflow ui shows one graph which is mixture of 2 functions. I would ideally like each to be seperate.
![decile_curve_artifact_on_data_churn](https://user-images.githubusercontent.com/48615547/183687070-4edf9cda-9d9f-4c73-84d2-7ac67bd74bb9.png)

this plot is the combination of my cummulative gain and the decile plot. I although have these 2 as seperate functions

**Comment by WeichenXu123** [comment, marked as answer]
Do you close the figure object after drawing one plot complete ?
If No then when you drawing next plot it will continue draw on the previous figure.

**Reply by khojarohan4** [reply]
I didnt close and left it running in the background. Closed all now and added a line to close the fig in each function I'm calling. Thank you so much. Saved me time.
